vssexporters/vss2binary.py

```diff
@@ -114,6 +114,9 @@ def export_node(node, generate_uuid, out_file):
         nodeuuid = node.uuid
     b_nodeuuid = nodeuuid.encode('utf-8')
 
+    # Export nodevalodate to binary
+    if node.validate != "":
```

Comment:
Before the changes I made, the vss2binary tool exports each one of the nodes using a set of attributes defined in the tool to a binary file with this structure: 
```mermaid
flowchart  LR
al[Number of Bytes of A]
a[Byte stream of A]
bl[Number of Bytes of B]
b[Byte stream of B]
  al-->a-->bl-->b-->....
```
In case any of the attributes defined in the tool is not present in the node (or is empty), the number of bytes when exporting is set to 0 and no byte stream is exported. Maybe @UlfBj can explain more about that.
The validate attribute was included in the set of attributes defined in the tool, but it was not checked if the node included it, so it was exported as 0:
```mermaid
flowchart  LR
al[Number of Bytes of A]
a[Byte stream of A]
bl[Number of Bytes of B]
b[Byte stream of B]
  al-->a-->0-->bl-->b-->....
```
The change I made in the vss2binary tool gets the value of the validate attribute in the node supposing it is a core attribute, but I am open to do it using the extended_attributes.

I have a problem using the extended_attributes which is the fact that if I include the validate attribute in the whitelist, it does not get included in the extended attributes list (and the whitelisted_extended_attributes of a node is converted to empty) when the vspec is loaded. There should be (I think) an error somewhere.

Reply:
see comment I made in #197 

Extended attributes "should" work at least until it reaches the exporter. If you can not get them to work, maybe try showing in detail how you called the tool (cmdline) and then how you tried to access the extended attribute.

In any  case, that of course is independent, from how the (non-)existence of the attribute shall be handled by the binary exporter. Still waiting for @UlfBj  for any insights 😄

Reply:
I do not think I have any deeper insights than what the two of you have already provided;). 

I think that adding support for this via the extended attribute is a good idea, as we then get tool support for the Access Control Selection as defined in VISSv2.

Reply:
I have modified the code to obtain the validate claim from extended attributes instead of having it in core attributes.
Now the the validate attribute is supported using the extended attribute flag. 
I think this is enough for us to be able to use the tool for VISSv2.